test/onchain/secp256k1/Secp256k1Arithmetic.t.sol

```diff
@@ -108,6 +110,54 @@ contract Secp256k1ArithmeticTest is Test {
         assertFalse(wrapper.isOnCurve(point));
     }
 
+    struct IsPointCase {
+        string P;
+        bool expected;
+    }
+
+    function testVectors_Point_isPoint_noble_curves() public {
```

Comment:
Not sure how the naming for this one. Maybe this should be put in the (De)Serialization part. But at the same time it test if the point is on the Curve.

Reply:
Made the rename for easier grepping with [59a03ce](https://github.com/verklegarden/crysol/pull/32/commits/59a03cecbee2b375c8d2a730cb0bf3a96eb0685d)